### DISCUSSION TITLE
ESP8266 and latest version

### DISCUSSION THREAD

**Discussion by rk125** [syssi/esphome-jbd-bms#73] [General]
To ones using esp8266 with this project. 
I had to recompile and reflash my esp8266 and it refused to work for me. Move was from v1.3.0 (compiled somewhere in fall 2022) to latest 1.5.0 now. Getting it on workbench and checking serial logs - esp was crashing right after startup and rebooting constantly without any obvious error. Even if I got it up somehow, was not responding to MQTT commands.
My battery is just 16 cells, so I took out cells 17-32 and also redundant temperature sensors from yaml file. After that all is back to normal.
Just wanted to let know, probably time to switch to ESP32 for bigger batteries. 
Btw, even recompiling now 1.3.0 did not help, it must be some other library taking more resources.

**Comment by syssi** [comment]
Yes...  unfortunately we are hitting the limits of an ESP8266 pretty easily. A ESP32 is recommended!
